tests/sentry/tasks/integrations/github/test_open_pr_comment.py

```diff
@@ -1102,43 +1076,6 @@ def test_comment_workflow_missing_pr(
             "github_open_pr_comment.error", tags={"type": "missing_pr"}
         )
 
-    @patch("sentry.tasks.integrations.github.open_pr_comment.metrics")
-    def test_comment_workflow_missing_org(
-        self,
-        mock_metrics,
-        _,
-        mock_safe_for_comment,
-        mock_issues,
-        mock_function_names,
-        mock_reverse_codemappings,
-        mock_pr_filenames,
-    ):
-        self.pr.organization_id = 0
-        self.pr.save()
-
-        open_pr_comment_workflow(self.pr.id)
-
-        assert not mock_pr_filenames.called
-        mock_metrics.incr.assert_called_with(
-            "github_open_pr_comment.error", tags={"type": "missing_org"}
-        )
-
-    def test_comment_workflow_missing_org_option(
```

Comment:
these tests can be removed because we were fetching the org in order to check the org option, but since we're no longer checking the org option we're also no longer fetching the org